src/core/agent/AgentService.ts

```diff
@@ -21,18 +26,24 @@ export default class AgentService {
     #agent?: Agent
     #agentLanguage?: Language
     #pubsub: PubSub
+    #tokens: Map<string, AuthInfo>
+    #tokenValidPeriod: number
+    #adminCredential: string
```

Comment:
I don't really get the idea/reason for why we have this variable `adminCredential`. I remember talking about printing a `*` capability token out to stdout so ad4min can get it from there. Then there should be no need for handling the 'admin case' in a special way within the executor, no?

Reply:
Yeah we talked about grep the output for the token, but after research on tauri side and more depth thinking, it's not a consistent design since only if we already generate an agent, it can further sign an jwt. And if the adminCredential is a jwt, it needs to refreshed periodically, this is not perfect for long running tasks.

In this new design, 
* the adminCredential gets passed in when starting ad4m-host, this is the practical way to run a binary with credentials suggested from [12 factor apps](https://12factor.net/config).
* restart the ad4min app, a new adminCredential generated.

Overall, I think jwt is good in authentication, but not fit for administration of long running tasks.